pkgs/development/python-modules/python-lz4/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, psutil
+, setuptools_scm
+, pkgconfig
+, pythonOlder
+, future
+}:
+
+buildPythonPackage rec {
+  pname = "python-lz4";
+  version = "2.1.0";
+
+  # get full repository inorder to run tests
+  src = fetchFromGitHub {
+    owner = "python-lz4";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "1vjfplj37jcw1mf8l810dv76dx0raia3ylgyfy7sfsb3g17brjq6";
+  };
+
+  buildInputs = [ setuptools_scm pkgconfig ];
```

Comment:
I think `pytestrunner` is missing here.

Reply:
Maybe you didn't encounter `distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')` because you have sandboxing turned off? You can turn it on with `nix build --option sandbox true` I think.

Reply:
Thank you for showing `--option sandbox true` yes I can reproduce the issue.